python/lancedb/remote/__init__.py

```diff
@@ -14,7 +14,7 @@
 import abc
 from typing import List, Optional
 
-import attr
```

Comment:
Do you need to update `pyproject.toml`?

Reply:
Already done in #508

Reply:
`attr` and `attrs` are functionality the same